src/script/util/DataDog.ts

```diff
@@ -65,6 +72,7 @@ export async function initializeDataDog(config: Configuration, domain?: string)
     forwardErrorsToLogs: true,
     forwardConsoleLogs: ['info', 'warn', 'error'], // For now those logs should be fine, we need to investigate if we need another logs in the future
     sessionSampleRate: 100,
+    silentMultipleInit: true,
```

Comment:
Let's remove this one, cause we would like to be warned in case we init it twice (this would be a mistake)
```suggestion
```